### ISSUE TITLE
the outputs are a vertical line

### ISSUE THREAD

**Issue by yangyang9706** [HasnainRaz/FC-DenseNet-TensorFlow#18]
I comment out the normalize function，but the outputs are just a vertical line and blank.

**Comment by HasnainRaz** [comment]
In the dataloader, under the resize image and mask function, use the "tf.image.resize_nearest_neighbor()" function for the mask. I assume its an issue with the bilinear interpolation in the resizing. Let me know if it works.
